main.js

```diff
@@ -0,0 +1,114 @@
+var startBtn = document.querySelector('.start');
+var taskInput = document.querySelector('.task-input');
+var task = document.querySelector('.task');
+var mins = document.querySelector('.mins');
+var secs = document.querySelector('.secs');
+var largeInt = 25;
```

Comment:
Rather than use big and small integers, think about how timers run in milliseconds so setting a 25 minute one would be 25 * 60 * 1000 and countdown from there. Then you can reformat that number into minutes and strings.